test/unit/test_galaxy_mapping.py

```diff
@@ -442,6 +442,21 @@ def expunge(cls):
         cls.model.session.expunge_all()
 
 
+class MockObjectStore(object):
+
+    def __init__(self):
+        pass
+
+    def size(self, dataset):
+        return 42
```

Comment:
The Answer to the Ultimate Question of Life, the Universe, and Everything, I guess :laughing: